tests/settings/ConfigurationGatheringTest.py

```diff
@@ -70,15 +70,16 @@ def test_gather_configuration(self):
 
         self.assertEqual(
             str(sections['cli']),
-            "cli {bears : 'JavaTestBear', config : " + repr(temporary) +
+            "cli {bears : 'JavaTestBear', config : " +
+            PathArg(repr(temporary)) +
             ", files : '*.java', save : 'True', test : '5'}")
 
         with make_temp() as temporary:
             sections, local_bears, global_bears, targets = (
                 gather_configuration(*args,
                                      arg_list=['-S test=5',
                                                '-f *.java',
-                                               '-c ' + escape(temporary, '\\'),
+                                               '-c ' + PathArg(temporary),
```

Comment:
Please just remove `escape` here. No `PathArg`. The latter gets auromatically applied by `DefaultArgParser`